src/cascade/executor/epiviz_runner.py

```diff
@@ -49,37 +61,73 @@ def execution_context_from_settings(settings):
         model_version_id=settings.model.model_version_id,
         model_title=settings.model.title,
         gbd_round_id=settings.gbd_round_id,
+        bundle_id=settings.model.bundle_id,
+        add_csmr_cause=settings.model.add_csmr_cause,
```

Comment:
add csmr cause seems like configuration? Let's lay these out and pick what goes where and _why._

Reply:
Yes, please.